anitya/lib/backends/__init__.py

```diff
@@ -265,6 +266,30 @@ def call_url(self, url, insecure=False):
 
             return requests.get(url, headers=headers, verify=not insecure)
 
+    @classmethod
+    def list_recent_projects(self, session):
+        return anitya.lib.model.Project.by_backend(session, self.name)
+
+    @classmethod
+    def update_projects(self):
+        session = anitya.app.SESSION
+        projects = self.list_recent_projects(session)
+        anitya.LOG.info(projects)
+        p = multiprocessing.Pool(anitya.app.APP.config.get('CRON_POOL', 10))
```

Comment:
Well, I simply reused code from current anitya_cron.py here. But yes, I can see cron job failures from time to time so different parallel execution module might suit better. I'll give it a try.